src/containers/Tenant/Tenant.tsx

```diff
@@ -78,10 +78,7 @@ export function Tenant(props: TenantProps) {
     const {PathType: currentPathType, PathSubType: currentPathSubType} =
         currentItem?.PathDescription?.Self || {};
 
-    let showBlockingError = false;
-    if (error && typeof error === 'object' && 'status' in error) {
-        showBlockingError = error.status === 403;
-    }
+    const showBlockingError = Boolean(error && !currentItem);
```

Comment:
`null` response could be for schema objects with problems, it doesn't mean, that the whole page is broken. For example, when I created issue https://github.com/ydb-platform/ydb-embedded-ui/issues/1052 I have the database, where I have two columns tables, that have `null` in schema response (sorry I didn't mentioned that). At the same tame, all other 100+ schema objects was just fine.

Now we have `PageError` even if one object is broken and I try to access it on first load (if I access it as result of navigation, it just displays "Schema is not available", it's that's needed